Framework/Crystal/src/PredictPeaks.cpp

```diff
@@ -101,9 +104,21 @@ void PredictPeaks::init() {
                   "values in the HKLPeaksWorkspace to the nearest integers if "
                   "checked.\n"
                   "Keep unchecked to use the original values");
+
   setPropertySettings("RoundHKL", make_unique<EnabledWhenProperty>(
                                       "HKLPeaksWorkspace", IS_NOT_DEFAULT));
 
+  // Build up a list of point groups to use
+  std::vector<std::string> prop2Options;
+  prop2Options.push_back("");
+  for (auto &pg : m_pointGroups) {
+    prop2Options.push_back(pg->getName());
+  }
+  declareProperty("PointGroup", "",
+                  boost::make_shared<StringListValidator>(prop2Options),
+                  "Which point group symmetry to apply to this crystal, "
+                  "reducing the number of expected HKL peaks?");
```

Comment:
I'm not sure I would be able to guess what this option really does from the description. What it does is not so straight forward, it will only find one peak for each family of reflections that are equivalent by point group symmetry.